support-frontend/assets/helpers/billingPeriods.js

```diff
@@ -6,7 +6,8 @@ const Quarterly: 'Quarterly' = 'Quarterly';
 const SixWeekly: 'SixWeekly' = 'SixWeekly';
 const weeklyBillingPeriods = [SixWeekly, Quarterly, Annual];
 export type BillingPeriod = typeof SixWeekly | typeof Annual | typeof Monthly | typeof Quarterly;
-export type DigitalBillingPeriod = typeof Monthly | typeof Annual;
+export type DigitalBillingPeriod = typeof Monthly | typeof Annual | typeof Quarterly;
```

Comment:
We can probably drop Quarterly from this definition now as it doesn't apply to standard DS

Reply:
Thanks, good point.